styles.css

```diff
@@ -411,3 +425,182 @@ form > ul {
 ::placeholder {
   color: #b3bac5;
 }
+
+.desktop-post {
+  display: none;
+}
+
+/* @media only screen and (max-width: 700px) { */
```

Comment:
- [ ] Kindly remove this commented piece of code to clean up the project and make it more professional ✨